pkg/sensors/tracing/kprobe_test.go

```diff
@@ -535,29 +533,14 @@ spec:
 }
 
 // __x64_sys_openat trace
-func getOpenatChecker() ec.MultiEventChecker {
-	kpChecker := ec.NewProcessKprobeChecker().
-		WithFunctionName(sm.Full("__x64_sys_openat")).
-		WithArgs(ec.NewKprobeArgumentListMatcher().
-			WithOperator(lc.Ordered).
-			WithValues(
-				ec.NewKprobeArgumentChecker().WithIntArg(-100),
-				ec.NewKprobeArgumentChecker().WithStringArg(sm.Full("/tmp/testfile")),
-				ec.NewKprobeArgumentChecker(),
-			))
-
-	return ec.NewUnorderedEventChecker(kpChecker)
-}
-
-// __x64_sys_openat trace
-func getOpenatMntChecker() ec.MultiEventChecker {
+func getOpenatChecker(dir string) ec.MultiEventChecker {
 	kpChecker := ec.NewProcessKprobeChecker().
 		WithFunctionName(sm.Full("__x64_sys_openat")).
 		WithArgs(ec.NewKprobeArgumentListMatcher().
 			WithOperator(lc.Ordered).
 			WithValues(
 				ec.NewKprobeArgumentChecker().WithIntArg(-100),
-				ec.NewKprobeArgumentChecker().WithStringArg(sm.Full(mountPath+"/testfile")),
+				ec.NewKprobeArgumentChecker().WithStringArg(sm.Full(filepath.Join(dir, "testfile"))),
```

Comment:
This will fail if someone passes a non-absolute path here. I would suggested using a suffix matcher here, possibly stripping the leading periods if they exist.

Reply:
Good catch! So this will fail in the case where we use `getOpenatChecker()` in a testcase where we don't use the correct path in `__x64_sys_openat`. I propose to just be aware of that (i.e. add a comment/issue) when we add new tests. 

I believe that we need more work to make that generic enough to handle all cases in `__x64_sys_openat`. WDYT?

Reply:
Documenting the limitation is a fine approach IMO.